src/test/ui/suggestions/issue-51055-missing-semicolon-between-call-and-tuple.stderr

```diff
@@ -0,0 +1,13 @@
+error[E0618]: expected function, found `bool`
+  --> $DIR/issue-51055-missing-semicolon-between-call-and-tuple.rs:14:5
+   |
+LL |       vindictive() //~ ERROR expected function, found `bool`
```

Comment:
It'd be nice if the main span here pointed at only `vindictive()`, given that that's where a function was expected (while also pointing at the entire span or the arguments, to make it clear what is happening.

Reply:
Yes, although I think this will require a bit of restructuring (need to assert that the expression is an `ExprCall` before you can get the callee span).

(regretfully, I'm out of rustc-dev time for at least a few days)